test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/*
+ * @test
+ * @bug 8081474
+ * @summary  Verifies if SwingWorker calls 'done'
+ *           before the 'doInBackground' is finished
+ * @run main TestDoneBeforeDoInBackground
+ */
```

Comment:
May I ask you to move the jtreg tags to the class declaration, please? When the file is opened in the IDE, they're not collapsed (together with the license block above) and easily accessible.

Reply:
Not sure on this..it seems it is same for other tests also..

Reply:
The majority of old tests have jtreg tags above imports. I find it very inconvenient.

Some newer tests have the jtreg tags before the class declaration.

jtreg accepts either variant.

When I review code, I always ask for this change because the jtreg tags are somewhat important and having them visible is just more convenient. The `@summary` tag provides the info similar to what one expects to find in a javadoc.